README.md

```diff
@@ -86,37 +86,60 @@ To Register an app:
 5. After the app is created, you'll land on your app management page. Take note of the **Application (client) ID** as this would be needed for the step 1B below.
 6. Click **Authentication**, and add new Redirect URI with type **Public client (mobile & desktop)**. Enter redirect URI in format: `msauth.<app.bundle.id>://auth`. Replace <app.bundle.id> with the **Bundle Identifier** for your application. 
 7. Hit the **Save** button in the top left, to save these updates.
+8. Click **Make this change for me** and then download the code sample for macOS
 
-## 2B: Configure your application
+## Step 3: Run the sample
+
+1. Click the Run Button in the top menu or go to Product from the menu tab and click Run.
+2. Once the sample app launches, click on the 'Call Microsoft Graph API' button to go through the sign in flow and see the results from Microsoft Graph.
 
-1. Configure your application defaults
+## How to add MSAL libarary into your existing Xcode project
+
+## Step 1: Configure your application Info.plist
+
+Add URI scheme in the  `Info.plist`. Redirect URI scheme follows the format `msauth.[app_bundle_id]`. Make sure to substitue [app_bundle_id] with the **Bundle Identifier** for your application.
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+  <dict>
+    <key>CFBundleURLSchemes</key>
+    <array>
+      <string>msauth.[app_bundle_id]</string>
+    </array>
+  </dict>
+</array>
+```
 
-In the `ViewControler.swift` file, update the `kClientID` variable with your Application (client) ID.
+## Step 2: Configure your application defaults
+
+In your app, add the `client` variable with your Application (client) ID.
 
 ```swift
-// Update the below to your client ID you received in the portal. The below is for running the demo only
-    
-let kClientID = "Enter_the_Application_Id_here"
+// For example, you can declare a client id in this way. Below ID is just an sample.
+    	
+let clientID = "66855f8a-60cd-445e-a9bb-8cd8eadbd3fa"
 ```
 
-In the `ViewControler.swift` file, update the `kAuthority` variable with your Azure AD and Microsoft Graph endpoints for your national cloud. For global access, use following values:
+In your app, add the `authority` variable with your Azure AD and Microsoft Graph endpoints for your national cloud. For global access, use following values:

```

Comment:
Should we just say "add variables with your Azure AD authority and Microsoft Graph endpoint for your national cloud."?